CMake/CMakeLibs.cmake

```diff
@@ -163,7 +163,12 @@ macro(ADD_OSQUERY_LIBRARY IS_CORE TARGET)
   if(${IS_CORE} OR NOT OSQUERY_BUILD_SDK_ONLY)
     add_library(${TARGET} OBJECT ${ARGN})
     add_dependencies(${TARGET} osquery_extensions)
-    SET_OSQUERY_COMPILE(${TARGET} "${CXX_COMPILE_FLAGS} -static")
+    # TODO(WIN64): For Win64, ignore the whole "static" thing for now
+    if(WIN32)
+      SET_OSQUERY_COMPILE(${TARGET} "${CXX_COMPILE_FLAGS}")
+    else()
+      SET_OSQUERY_COMPILE(${TARGET} "${CXX_COMPILE_FLAGS} -static")
```

Comment:
I'm not sure if `set_target_properties` overwrites the (in this case) `COMPILE_FLAGS` property every subsequent call (https://github.com/facebook/osquery/blob/master/CMake/CMakeLibs.cmake#L12) but if it doesn't this could be a simple:
```cmake
SET_OSQUERY_COMPILE(${TARGET} "${CXX_COMPILE_FLAGS}")
if(NOT WIN32)
  SET_OSQUERY_COMPILE(${TARGET} "-static")
endif()
```